### PR TITLE
Fix: Changed  statement like while to enhanced for loop and `if` to ternary operator

### PR DESCRIPTION
**Pull request by mdxabu**
Dear Maintainers,

I wanted to inform you that I have made some modifications in two files: `ContainersListElements.java` and `ContainerDetails.java`. Specifically, I have changed the `while` loop to an `enhanced for` loop and altered the `if` statement to a `ternary` statement.

Best regards,
F Mohamed Abdullah

### CODE REVIEW

File: src/main/java/io/fabric8/maven/docker/model/ContainerDetails.java [hunk 1]
```diff
@@ -77,10 +77,7 @@ public Map<String, String> getLabels() {
     public String getName() {
         String name = json.get(NAME).getAsString();
 
-        if (name.startsWith(SLASH)) {
-            name = name.substring(1);
-        }
-        return name;
+        return name.startsWith(SLASH)?name.substring(1):name;
```
Comment (rohanKanojia) [inline src/main/java/io/fabric8/maven/docker/model/ContainerDetails.java:80]: I find if statements more readable than ternary operators.

If you're using ternary here, please add a bit formatting so that it's more readable
```suggestion
        return name.startsWith(SLASH) ? name.substring(1) :
          name;
```

File: src/main/java/io/fabric8/maven/docker/model/ContainersListElement.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
 
 import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
+import io.fabric8.maven.docker.access.ContainerHostConfig;
```
Comment (rohanKanojia) [inline src/main/java/io/fabric8/maven/docker/model/ContainersListElement.java:10]: why is this import needed?

Comment (mdxabu) [reply src/main/java/io/fabric8/maven/docker/model/ContainersListElement.java:10]: No, I forgot to remove it.

File: src/main/java/io/fabric8/maven/docker/model/ContainerDetails.java [hunk 3]
```diff
@@ -1,17 +1,12 @@
...
-import java.util.Map;
-import java.util.Set;
-
 import com.google.common.base.Joiner;
-import com.google.common.base.Preconditions;
 import com.google.gson.JsonArray;
 import com.google.gson.JsonObject;
 
+import java.time.Instant;
+import java.util.*;
```
Comment (rohanKanojia) [inline src/main/java/io/fabric8/maven/docker/model/ContainerDetails.java:8]: Could you please revert these import changes?

Please [configure your IDE to not add wildcard imports](https://stackoverflow.com/a/25916603/2830116)

**Review by rohanKanojia — APPROVED** [review]

**Comment by rohanKanojia** [comment]
Thanks!
